[Turnstile] Document that `action` can only contain alphanumeric, `-` and `_` chars (#6249)
cloudflare/cloudflare-docs: content/turnstile/get-started/client-side-rendering.md

| JavaScript Render Parameters | Data Attribute | Description |
| --- | --- | --- |
| `sitekey` | `data-sitekey` | Every widget has a sitekey. This sitekey is associated with the corresponding widget configuration and is created upon the widget creation. |

> | `action` | `data-action` | A customer value that can be used to differentiate widgets under the same sitekey in analytics and which is returned upon validation. This should only contain alphanumeric characters, `_` and `-`. |

Reviewer: ```suggestion
| `action` | `data-action` | A customer value that can be used to differentiate widgets under the same sitekey in analytics and which is returned upon validation. This can only contain up to 32 lowercase alphanumeric characters, `_` and `-`. |
```
Turnstile staging has added a validation regex for action: `/^[a-z0-9_-]{0,32}$/i`

> | `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. |

Reviewer: ```suggestion
| `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters, `_` and `-`.  |
```
Turnstile staging has added a validation regex for cData: `/^[a-z0-9_-]{0,255}$/i`

> | `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters, `_` and `-`.  |

Reviewer 2: ```suggestion
| `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters, `_`, and `-`.  |
```

> | `action` | `data-action` | A customer value that can be used to differentiate widgets under the same sitekey in analytics and which is returned upon validation. This can only contain up to 32 lowercase alphanumeric characters, `_` and `-`. |
> | `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters, `_` and `-`.  |

Reviewer: ```suggestion
| `action` | `data-action` | A customer value that can be used to differentiate widgets under the same sitekey in analytics and which is returned upon validation. This can only contain up to 32 lowercase alphanumeric characters including `_` and `-`. |
| `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters including `_`, and `-`.  |
```
imo sounds a bit better and makes it clear than `_` and `-` are also in the 32/255 limit

> | `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters including `_`, and `-`.  |

Reviewer 2: ```suggestion
| `cData` | `data-cdata` | A customer payload that can be used to attach customer data to the challenge throughout its issuance and which is returned upon validation. This can only contain up to 255 lowercase alphanumeric characters including `_` and `-`.  |
```
